### PR TITLE
Add missing info for `change` column and table comment in Guides

### PR DESCRIPTION
**Pull request by gustiando**
### Summary

Some people (myself included) rely heavier on the guides for a quick reference more than the api docs. Thought I'd throw this one in there to see if it helps.

> You can help improve the Rails guides or the API reference by making them more coherent, consistent, or readable, **adding missing information**, correcting factual errors, fixing typos, or bringing them up to date with the latest edge Rails.

This will address the "adding missing information" suggestion from the [guide](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#contributing-to-the-rails-documentation).

### CODE REVIEW

File: guides/source/active_record_migrations.md [hunk 1]
```diff
@@ -612,10 +612,12 @@ definitions:
 * [`add_index`][]
 * [`add_reference`][]
 * [`add_timestamps`][]
+* [`change_column_comment`][] (must supply a `:from` and `:to` option)
 * [`change_column_default`][] (must supply a `:from` and `:to` option)
 * [`change_column_null`][]
 * [`create_join_table`][]
 * [`create_table`][]
+* [`change_table_comment`][] (must supply a `:from` and `:to` option)
```
Comment (jonathanhefner) [inline guides/source/active_record_migrations.md:620]: What about bumping up `change_table_comment` to preserve alphabetical order?

```suggestion
* [`change_column_null`][]
* [`change_table_comment`][] (must supply a `:from` and `:to` option)
* [`create_join_table`][]
* [`create_table`][]
```

File: guides/source/active_record_migrations.md [hunk 2]
```diff
@@ -612,10 +612,12 @@ definitions:
 * [`add_index`][]
 * [`add_reference`][]
 * [`add_timestamps`][]
+* [`change_column_comment`][] (must supply a `:from` and `:to` option)
```
Comment (jonathanhefner) [inline guides/source/active_record_migrations.md:615]: To use the reference-style link syntax (`[link text][]`), both `change_column_comment` and `change_table_comment` need their URLs defined elsewhere.  Let's add them to the block of URLs on lines 648-658.

**Review by jonathanhefner — COMMENTED** [review]
I think it makes sense to add these.  Though it might be worth modifying "the `change` method supports only these migration definitions" to sound less final, in case of future missing entries like these.

When you are happy with the changes, would you mind [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?

File: guides/source/active_record_migrations.md (hunk 2 above)
Comment (gumatias) [reply guides/source/active_record_migrations.md:614]: Thanks for the guidance @jonathanhefner . I'm happier with the changes now.

**Comment by gumatias** [comment]
Thanks for the guidance @jonathanhefner . I'm happier with the changes now.

**Comment by jonathanhefner** [comment]
> I was considering linking to the [api docs](https://api.rubyonrails.org/v7.0.2.3/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html) or something. But am curious what you think. I tried to avoid blowing up the scope of this change too much.

Do you mean adding another link, in addition to the individual method links?  I think it would be nice to do so, but there doesn't seem to be a definitive place to link to.  Linking to [`SchemaStatements`](https://api.rubyonrails.org/v7.0/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html) for its list of methods doesn't seem quite right because there are additional methods listed there.  Adding some documentation to the `SchemaStatements` module itself might improve that, but, I agree, that would be out of scope for this PR.

File: guides/source/active_record_migrations.md [hunk 3]
```diff
@@ -604,16 +604,17 @@ and
 
 The `change` method is the primary way of writing migrations. It works for the
 majority of cases, where Active Record knows how to reverse the migration
-automatically. Currently, the `change` method supports only these migration
-definitions:
+automatically. Below are just some of the migration definitions currently supported:
```
[the remark this answers is not in the document]
Comment (gumatias) [reply guides/source/active_record_migrations.md:607]: Thanks! makes sense.

**Comment by gumatias** [comment]
@jonathanhefner yes, that's what I had in mind.

**Comment by jonathanhefner** [comment]
Thank you, @gumatias! :tada:

**Pull request merged**